src/service.py

```diff
@@ -137,6 +130,14 @@ def get_search_create_study(search_space_json, operation):
     if operation == "EXP_TRIAL_GENERATE_NEW":
         experiment_name, total_trials, parallel_trials, direction, hpo_algo_impl, id_, objective_function, tunables, value_type = get_all_tunables(
             search_space_json)
+        if(not total_trials):
+            return -1
+        if(not parallel_trials):
+            parallel_trials = n_jobs
```

Comment:
1. Log an error message here if the total_trials is not set
2. parallel trials cannot be anything other than "1", log an error message for that as well
3. Space after if. ` if ()`